### PR TITLE
update learning objectives for prep and sprint 1 projects

### PR DESCRIPTION
**Pull request by SallyMcGrath**
testable actions
servers and databases

understand is a banned word as it's a state of being not an act of doing
multiple servers

formatting

docker cloud

grPC

### CODE REVIEW

File: projects/http-auth/README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ Timebox: 6 days
 
 ## Learning objectives:
 
-- Use Go's net/http package to build start a simple server that responds to local requests
-- Get to know HTTP GET and response codes
-- Get familiar with cURL
+- Use Go’s net/http package to build start a simple server that responds to local requests
```
Comment (illicitonion) [inline projects/http-auth/README.md:13]:
```suggestion
- Use Go's net/http package to build start a server that responds to local requests
```

File: projects/http-auth/README.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ Timebox: 6 days
...
+- Use HTTP GET and response codes
+- Transfer data with cURL
```
Comment (illicitonion) [inline projects/http-auth/README.md:15]:
```suggestion
- Make HTTP GET requests using cURL
- Respond to HTTP get requests with relevant response codes from a Go server
```

File: projects/multiple-servers/README.md [hunk 3]
```diff
@@ -10,13 +10,15 @@ Timebox: 5 days
...
+- Describe microservices
+- Separate concerns of microservices
+- Replicate a local setup in the cloud on a virtual machine
 - Configure nginx to talk to 2-3 copies of the API server
-- Some web security ideas (CORS)
-- Reverse proxy configuration, routing on path
-- Health checks
+- Resolve CORS issues
+- Configure a reverse proxy, routing on path
+- Conduct server health checks
```
Comment (illicitonion) [inline projects/multiple-servers/README.md:19]: I'm not 100% on this one... While we have the trainee observe nginx marking a server as bad, I don't think we have them _conduct_ health checks, and we don't really talk about how the health checks happen or how servers are marked as healthy again...

I'd maybe remove this LO, or if we want to keep it, we should maybe expand the last section a little...

**Review by illicitonion — APPROVED** [review]

File: projects/multiple-servers/README.md (hunk 3 above)
Comment (SallyMcGrath) [reply projects/multiple-servers/README.md:19]: remove it for now I think
```suggestion

```

**Review by SallyMcGrath — COMMENTED** [review]
really just seeing if I can suggest this line change to myself

**Pull request merged**